app/assets/stylesheets/application/member-card.scss

```diff
@@ -1,16 +1,26 @@
+.column--cards {
+  display: flex;
+  flex-direction: column;
+}
+
 .to-member-card {
   & {
     margin: 16px 0;
     width: 100%;
     max-width: 600px;
-    height: 178px;
+    min-height: 178px;
     background-color: white;
     box-shadow: 0 0 3px $palette-grey;
-    display: inline-block;
+    border-radius: 0.3rem;
```

Comment:
I would use px instead of rem since rem varies depending on font-size and I don't think we want this to change based on that.